snow/consensus/snowman/snowmantest/block.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package snowmantest
+
+import (
+	"cmp"
+	"context"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow/choices"
+	"github.com/ava-labs/avalanchego/utils"
+)
+
+const (
+	GenesisHeight        uint64 = 0
+	GenesisUnixTimestamp int64  = 1
+)
+
+var (
+	_ utils.Sortable[*Block] = (*Block)(nil)
+
+	GenesisID        = ids.GenerateTestID()
+	GenesisTimestamp = time.Unix(GenesisUnixTimestamp, 0)
+	GenesisBytes     = GenesisID[:]
```

Comment:
We don't _need_ these values.. I think they are maybe slightly nicer to use rather than doing `snowtest.Genesis.IDV`... And using `snowtest.Genesis.ID()` is slightly annoying in tests during debugging (for myself).

Fwiw - `snowtest.GenesisID == snowtest.Genesis.ID()`